app/containers/Forms/Form.tsx

```diff
@@ -29,6 +31,12 @@ interface Props {
 
 const CUSTOM_FIELDS = {
   fuel: props => <FuelFields query={props.formContext.query} {...props} />,
+  emissionSource: props => (
+    <EmissionSourceFields query={props.formContext.query} {...props} />
+  ),
+  emissionGas: props => (
+    <EmissionGasFields query={props.formContext.query} {...props} />
+  ),
```

Comment:
Those are not fragment containers, no need to set the query prop. `emissionSource: EmissionSourceFields` should be enough